src/main/cpp/Robot.cpp

```diff
@@ -234,7 +259,7 @@ void Robot::RobotPeriodic() {
         RunShooterSM();
     }
 
-    units::radian_t turretHeadingInGlobal{
+    /*units::radian_t turretHeadingInGlobal{
```

Comment:
Use `#if 0` and `#endif` to comment this code out since there's so much, and leave a comment explaining why it was commented out.